app/operations/organizations/destroy.rb

```diff
@@ -0,0 +1,12 @@
+module Organizations
+  class Destroy < Operation
+    integer :id
+
+    def execute
+      Organization.transaction do
+        organization = Organization.find(id)
+        Activation.disable_instances!([organization])
```

Comment:
the activation module just adds a wrapper around calling `.disable!` and is generalized in the `DeactivatableResource` module, you could just do `Organization.find(id).disable!` here instead.